cmd/run.go

```diff
@@ -34,7 +35,7 @@ var (
 )
 
 func init() {
-	runCmd.Flags().StringSliceVarP(&nodeIPs, "node-ips", "n", []string{}, "List of remote hosts")
+	runCmd.Flags().StringSliceVarP(&nodeIPs, "node-ips", "n", []string{}, "List of remote hosts (comma-delimited)")
```

Comment:
Since we're changing the syntax to support `x.x.x.x[:y.y.y.y]`, should we reflect that in the help message?